admin_guide/manage_scc.adoc

```diff
@@ -208,6 +195,77 @@ In order to preserve customized SCCs during upgrades, do not edit settings on
 the default SCCs other than priority, users, and groups.
 ====
 
+[[example-security-context-constraints]]
+=== Example Security Context Constraints Settings
+
+.Without Explicit runAsUser Setting
+----
+apiVersion: v1
+kind: Pod
+...
+fsGroup:
+  type:
+  ranges:
+  - min: 5000
+    max: 6000
+runAsUser: <1>
+  type:
+  uidRangeMin:
+  uidRangeMax:
+seLinuxContext:
+  type:
+  SELinuxOptions:
+    user: <selinux-user-name>
+    role: ...
+    type: ...
+    level: ...
+supplementalGroups:
+  type: MustRunAs
+  ranges:
+  - min: 5000
+    max: 6000
+----
+<1> When `runAsUser` is not explicitly set, one of the available SCCs (most likely
+ a restricted SCC) will process the configuration. `runAsUser` will be set to a
+ random value.  The admission controller looks for the
+ `openshift.io/sa.scc.uid-range` annotation to populate range fields. See
+ xref:../architecture/additional_concepts/authorization.adoc#understanding-pre-allocated-values-and-security-context-constraints[Understanding
+ Pre-allocated Values and Security Context Constraints] for more information.
```

Comment:
I'd suggest another description for that field:

> When a container (or a pod) doesn't request a user id under which it should be run upon, the effective UID depends on the SCC that admits this pod. Because `restricted` SCC is granted to all authenticated users by default, it will be available to all users and service accounts and will be used in most of the cases. The `restricted` SCC uses `MustRunAsRange` strategy for constraining and defaulting the possible values of the `securityContext.runAsUser` field. As it doesn't provide this range, the admission controller will look for the `openshift.io/sa.scc.uid-range` annotation on the current namespace to populate range fields. At the end, a container will have `runAsUser` equal to a semi-random value (the first value of the range that is hard to predict because every namespace has different ranges).

@ahardin-rh Could you review this description and improve it? The link to "Understanding Pre-allocated Values and Security Context Constraints" also should be in this paragraph.